### ISSUE TITLE
Helper function to pull unique lists

### ISSUE THREAD

**Issue by peterdesmet** [inbo/etn#183]
I often want a unique list of items from my selection, e.g. get the tag_serial_numbers associated with my animals:

```R
tag_serial_numbers <- get_animals() %>% distinct(tag_serial_number) %>% pull() # a vector
```

I can use that vector to do:

```R
get_acoustic_tags(tag_serial_number = tag_serial_numbers)
```

@jreubens @PieterjanVerhelst @damianooldoni Would it be useful to have a helper function for this:

```R
get_animals() %>% extract(tag_serial_number) # a vector
```

Note that this function could have build-in functionality to deal with multiple comma separated values:

```
"A,B", "C" -> c("A", "B", "C")
```

**Comment by PieterjanVerhelst** [comment]
I think this is highly relevant and was actually something I was missing a bit already. Although you can get those info by working around it as suggested, this is much more elegant. Another example is getting receiver info for the receivers involved in a deployment-selection.

**Comment by damianooldoni** [comment]
I think it's quite good to have, yes. It should not be a big function. I can write it for you if you want.

**Comment by peterdesmet** [comment]
@damianooldoni Yes, please. How should we name it without clashing with base or dplyr functions?

- `extract()`: generic short name
- `pull_distinct()`: verb first
- `distinct_pull()`: what actually happens
- `distinct_vector()`: what is returned
- `list_values()`: similar to other `list_` functions already in ETN who also return a unique vector of elements

I think `list_values()` is the best one.

**Comment by PieterjanVerhelst** [comment]
Definitely `list_values()` to keep it straightforward with those other functions.

**Comment by damianooldoni** [comment]
Yes, I vote for `list_values()` too.

**Comment by jreubens** [comment]
me too of list _values and yes this is a functionality that we warmly welcome :-)

**Comment by damianooldoni** [comment]
Expect a PR in a moment.

**Comment by peterdesmet** [comment]
Implemented in `sql_views` branch via #190.
